src/client/common/utils/localize.ts

```diff
@@ -975,6 +975,11 @@ export namespace DataScience {
         'DataScience.showDataViewerFail',
         'Failed to create the Data Viewer. Check the Jupyter tab of the Output window for more info.'
     );
+
+    export const kernelTipMessage = localize(
+        'DataScience.kernelTipMessage',
+        'Tip: You can change your notebook kernel in the status bar.'
```

Comment:
The messages are different here and in package.nls.json

Reply:
Yeah doesn't really matter. The package.nls.json is source of truth.